2022-01-21 17:05:25,972 INFO    MainThread:4187 [wandb_setup.py:_flush():71] setting env: {}
2022-01-21 17:05:25,973 INFO    MainThread:4187 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-21 17:05:25,973 INFO    MainThread:4187 [wandb_init.py:_log_setup():371] Logging user logs to /Users/gdegournay/PycharmProjects/Udacity/MLOps/MLOps_ex/reproducible_workflow/exercices/lesson-1-machine-learning-pipelines/exercises/exercise_1/starter/wandb/run-20220121_170525-cj985dhe/logs/debug.log
2022-01-21 17:05:25,973 INFO    MainThread:4187 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/gdegournay/PycharmProjects/Udacity/MLOps/MLOps_ex/reproducible_workflow/exercices/lesson-1-machine-learning-pipelines/exercises/exercise_1/starter/wandb/run-20220121_170525-cj985dhe/logs/debug-internal.log
2022-01-21 17:05:25,973 INFO    MainThread:4187 [wandb_init.py:init():404] calling init triggers
2022-01-21 17:05:25,973 INFO    MainThread:4187 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-21 17:05:25,973 INFO    MainThread:4187 [wandb_init.py:init():460] starting backend
2022-01-21 17:05:25,973 INFO    MainThread:4187 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-01-21 17:05:25,996 INFO    MainThread:4187 [backend.py:ensure_launched():216] starting backend process...
2022-01-21 17:05:26,644 INFO    MainThread:4187 [backend.py:ensure_launched():221] started backend process with pid: 4200
2022-01-21 17:05:26,645 INFO    MainThread:4187 [wandb_init.py:init():469] backend started and connected
2022-01-21 17:05:26,648 INFO    MainThread:4187 [wandb_init.py:init():533] updated telemetry
2022-01-21 17:05:26,650 INFO    MainThread:4187 [wandb_init.py:init():563] communicating current version
2022-01-21 17:05:28,222 INFO    MainThread:4187 [wandb_init.py:init():568] got version response 
2022-01-21 17:05:28,222 INFO    MainThread:4187 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-21 17:05:28,507 INFO    MainThread:4187 [wandb_init.py:init():606] starting run threads in backend
2022-01-21 17:05:32,452 INFO    MainThread:4187 [wandb_run.py:_console_start():1810] atexit reg
2022-01-21 17:05:32,452 INFO    MainThread:4187 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-21 17:05:32,453 INFO    MainThread:4187 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-21 17:05:32,455 INFO    MainThread:4187 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-21 17:05:32,455 INFO    MainThread:4187 [wandb_init.py:init():633] run started, returning control to user process
2022-01-21 17:06:12,665 INFO    MainThread:4187 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-21 17:06:12,666 INFO    MainThread:4187 [wandb_run.py:_restore():1752] restore
2022-01-21 17:06:15,160 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 638
  total_bytes: 638
}

2022-01-21 17:06:15,559 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 638
  total_bytes: 638
}

2022-01-21 17:06:15,691 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 638
  total_bytes: 5300
}

2022-01-21 17:06:15,792 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 638
  total_bytes: 12059
}

2022-01-21 17:06:15,899 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 638
  total_bytes: 12059
}

2022-01-21 17:06:16,004 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 638
  total_bytes: 12059
}

2022-01-21 17:06:16,106 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 638
  total_bytes: 12059
}

2022-01-21 17:06:16,210 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12059
  total_bytes: 12059
}

2022-01-21 17:06:16,315 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12059
  total_bytes: 12059
}

2022-01-21 17:06:16,418 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12059
  total_bytes: 12059
}

2022-01-21 17:06:16,520 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12059
  total_bytes: 12059
}

2022-01-21 17:06:16,624 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12059
  total_bytes: 12059
}

2022-01-21 17:06:16,731 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12059
  total_bytes: 12059
}

2022-01-21 17:06:17,480 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12059
  total_bytes: 12059
}

2022-01-21 17:06:18,118 INFO    MainThread:4187 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12059
  total_bytes: 12059
}
local_info {
}

2022-01-21 17:06:19,598 INFO    MainThread:4187 [wandb_run.py:_append_files():2180] logging synced files
